app/components/IntlGlobalProvider/index.js

```diff
@@ -5,6 +5,10 @@ import { useIntl } from 'react-intl';
 // 'intl' service singleton reference
 let intl;
 
+export const setIntl = i => {
```

Comment:
please pass in the created instance rather than the function that would create it to setIntl